common/src/gas_provider/node.rs

```diff
@@ -268,17 +268,19 @@ impl<ExternalId: Clone, Id: Clone + Copy, Balance: Zero + Clone + Copy>
     }
 
     /// Returns whether the node is of `ReservedLocal` type
-    pub(crate) fn is_reserved_local(&self) -> bool {
-        matches!(self, Self::ReservedLocal { .. })
+    pub(crate) fn is_cut(&self) -> bool {
+        matches!(self, Self::Cut { .. })
     }
 
-    /// Returns wether the node is of `Reserve` type
+    /// Returns whether the node is of `Reserve` type
     pub(crate) fn is_reserved(&self) -> bool {
         matches!(self, Self::Reserved { .. })
     }
 
-    pub(crate) fn is_orphan(&self) -> bool {
-        self.is_reserved() | self.is_reserved_local()
+    /// Returns whether the node is independent and isn't part of the tree structure
+    /// (not node's parent, not node's child).
+    pub(crate) fn is_independent(&self) -> bool {
```

Comment:
what do you think of `is_separate`? Sounds more relevant for me. Can keep independent.